Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/mtl_command_buffer.mm

```diff
@@ -1742,6 +1750,22 @@ inline void PopDebugGroupCmd(id<MTLRenderCommandEncoder> encoder, IntermediateCo
 
     mStateCache.scissorRect = clampedRect;
 
+    if (map)
+    {
+        auto adjustedOrigin =
+            [map mapPhysicalToScreenCoordinates:MTLCoordinate2DMake(clampedRect.x, clampedRect.y)
+                                       forLayer:0];
```

Comment:
Since forLayer's hardcoded to 0 here, too, it looks like this only works for framebuffers with a single color attachment.